jetty-util/src/main/java/org/eclipse/jetty/util/thread/QueuedThreadPool.java

```diff
@@ -425,16 +424,34 @@ public int getMinThreads()
      * @return number of reserved threads or or -1 for heuristically determined
      * @see #setReservedThreads
      */
-    @ManagedAttribute("the number of reserved threads in the pool")
+    @ManagedAttribute("number of configured reserved threads")
```

Comment:
Should we deprecate this?  It really is the same as maxReserved threads.  Or should we just change it to be the available reserved threads.